src/main/java/com/conveyal/gtfs/loader/DateField.java

```diff
@@ -0,0 +1,56 @@
+package com.conveyal.gtfs.loader;
+
+import com.conveyal.gtfs.storage.StorageException;
+
+import java.sql.JDBCType;
+import java.sql.PreparedStatement;
+import java.sql.SQLType;
+
+/**
+ * A GTFS date in the numeric format YYYYMMDD
+ */
+public class DateField extends Field {
+
+    public DateField (String name, Requirement requirement) {
+        super(name, requirement);
+    }
+
+    private int validate (String string) {
+        if (string.length() != 8) {
+            throw new StorageException("Date field should be exactly 8 characters long.");
+        }
+        int year = Integer.parseInt(string.substring(0, 4));
```

Comment:
It would probably be better to use a date parser for this.  That way we'd also find problems such as trying to parse the date February 30th.

Reply:
I changed this to use a LocalDate parser and added a test to check that it works right.